src/components/Page/Dependency/ApplicabilityEvidence.tsx

```diff
@@ -37,6 +38,21 @@ export default function ApplicabilityEvidence(props: Props): JSX.Element {
 						</div>
 						<Divider />
 					</>
+				) : (
+					// If CVE is not applicable
+					<>
+						<h6 className={css.subtitle}>Why is this CVE not applicable?</h6>
+						<div>
+							<div className={css.rowList}>
+								{props.data.evidence?.map((evidence, i) => (
+									<div key={i}>
+										<Row title="Reason" data={evidence.reason} />
```

Comment:
Are we only showing reasons when not applicable?

Reply:
yes. See https://jfrog-int.atlassian.net/browse/XRAY-81967 image for non-applicable